telco-ran/configuration/argocd/example/policygentemplates/example-multinode-reboot.yaml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Are 2 copies of the policy required? The multinode requires the user to fill in the proper MCP. Similarly the SNO version has 'master' already filled in (but this is the only valid MCP). Can we have just 1 copy where the user is instructed to fill in the correct MCP?  Same above for the PolicyGenerator version.